### PR TITLE
Flag WHM support for 5.4

### PR DESCRIPTION
**Pull request by Shirk**
No changes to WHM between 5.3 and 5.4 so it should be safe to mark it supported.

### CODE REVIEW

File: src/parser/jobs/whm/index.js [hunk 1]
```diff
@@ -22,14 +22,19 @@ export default new Meta({
...
 	contributors: [
 		{user: CONTRIBUTORS.NIV, role: ROLES.MAINTAINER},
 		{user: CONTRIBUTORS.VULCWEN, role: ROLES.DEVELOPER},
 		{user: CONTRIBUTORS.SUSHIROU, role: ROLES.DEVELOPER},
 	],
 	changelog: [
+		{
+			date: new Date('2021-01-08'),
+			Changes: () => <>Add 5.4 support for WHM</>,
+			contributors: [CONTRIBUTORS.NONO],
```
Comment (ayyaruq) [inline src/parser/jobs/whm/index.js:36]: Why did you mark this as Nono's work?

**Review by ayyaruq — CHANGES_REQUESTED** [review]
Thanks for the PR, please add yourself as a contributor and set it appropriately.

**Comment by ackwell** [comment]
(also a side note that merging this basically means you're on line for any support that comes up for whm until the next patch break point - make sure you drop by the discord server!

File: src/parser/jobs/whm/index.js (hunk 1 above)
Comment (Shirk) [reply src/parser/jobs/whm/index.js]: Copy paste was too strong.. fixed

**Review by ayyaruq — APPROVED** [review]
Thanks!

**Pull request merged**